lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -74,9 +74,9 @@ $folder.RegisterTaskDefinition($task_name, $task, 6, $username, $pass_to_use, $l
 $registered_task = $folder.GetTask("\$task_name")
 $registered_task.Run($null) | Out-Null
 
-$timeout = 10
+$registered_timeout = <%= registered_timeout %>
```

Comment:
Have you found the default `10` to be problematic? If not I'd just stick with the `execution_timeout`.

Reply:
Ok, we don't have any problem with this timeout. I remove this change.

Reply:
Done